### ISSUE TITLE
Cannot get build for Android versions smaller than 24.

### ISSUE THREAD

**Issue by ufktvfk** [mapbox/mapbox-unity-sdk#982]
Hi,   I cannot build apk for Android versions smaller than 24. In known issues page, it is said that minimum Android version supported is 15.

* Unity version:  2018.1.5f1
* Mapbox SDK version: 1.4.4
* The platform you're building to: Android
* Steps to recreate the bug if appropriate: Try to get build with Android version smaller than 24.
* Link:  https://www.mapbox.com/mapbox-unity-sdk/docs/02-known-issues.html

**Comment by jordy-isaac** [comment]
That's our custom native Android GPS Plugin. Its min requirement is set to 24 as it uses methods that are not available on earlier versions.

However, if you do want to downgrade you can download the Zip files without the command line: https://github.com/mapbox/mapbox-unity-sdk/releases

Some other things to try:

1. Uncheck `Android` in the inspector for
```Assets/Mapbox/Unity/Location/Plugins/Android/com.mapbox.android.unity-debug```


2. Delete whole folder:
```Assets/Mapbox/Unity/Location/Plugins/Android/```

If the above does not work I would suggest moving your min supported device up to lvl 24 to be able to make use of the Maps SDK for Unity.

**Comment by ufktvfk** [comment]
Which GPS methods you use in your native Android GPS Plugin since our app heaviliy uses GPS for world calibration so we may really need advanced GPS methods. Meantime, tusing min 24 means that we may lose lot of device models..

**Comment by ufktvfk** [comment]
I found a workaround to this issue but not sure it is correct or suggested way. I basically edited AndroidManifest file for mapbox unity sdk.

![mapbox package](https://user-images.githubusercontent.com/23186662/45226728-5be90000-b2c8-11e8-8b07-928e79e4388f.JPG)

After I did this, I successfully got build for API level 19 which is below 24. It worked fine in the phone.

Is this method can be trusted ?
